koku/api/settings/test/aws_category_keys/test_view.py

```diff
@@ -64,3 +65,56 @@ def test_put_bad_request(self):
         data = {"ids": ["bad_uuid", FAKE.uuid4()]}
         response = client.put(url, data, format="json", **self.headers)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+
+class SettingsAWSCategoryRBACTest(TestAwsCategoryClass):
+    """Test case for RBAC permissions to access settings."""
+
+    NO_ACCESS = {"aws.account": {"read": ["*"]}, "aws.organizational_unit": {"read": ["*"]}}
+    READ = {"settings": {"read": ["*"]}}
+    WRITE = {"settings": {"write": ["*"]}}
+    READ_WRITE = {"settings": {"read": ["*"], "write": ["*"]}}
```

Comment:
These are local variables not globals, so it may make more sense to keep them as lowercase.

Reply:
Valid point, let me change this.